src/app/client/src/app/modules/program-dashboard/pipes/chartType/chart-type.pipe.ts

```diff
@@ -11,6 +11,12 @@ export class ChartTypePipe implements PipeTransform {
         values: _.compact(chartData),
       });
     } else {
+      if(_.get(chartData, "filters")){
```

Comment:
what will happen with this if the filters is not there?

Reply:
Filters are mandatory for our use case here in the configuration still if they're not present this condition will not apply and other items will be returned as per the code